google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -394,7 +402,11 @@ public GapicSpannerRpc(final SpannerOptions options) {
                     .setTransportChannelProvider(channelProvider)
                     .setCredentialsProvider(credentialsProvider)
                     .setStreamWatchdogProvider(watchdogProvider)
-                    .setTracerFactory(options.getApiTracerFactory())
+                    .setTracerFactory(
+                        options.getApiTracerFactory(
+                            isDirectPathChannelCreated,
+                            false,
+                            isEmulatorEnabled(options, emulatorHost)))
```

Comment:
nit, here and for the other calls to this method:
```suggestion
                    .setTracerFactory(
                        options.getApiTracerFactory(
                            isDirectPathChannelCreated,
                            /* isAdminClient = */ false,
                            isEmulatorEnabled(options, emulatorHost)))
```